README.md

```diff
@@ -38,73 +27,62 @@ Additionally, it uses `slither` for static analysis.
 
 [Install slither](https://github.com/crytic/slither?tab=readme-ov-file#how-to-install)
 
-### Install
-
-As the project uses `foundry`, update the dependencies by running:
-
-```shell
-forge update
-```
+[Install node](https://nodejs.org/)
 
-Build the contracts:
+[Install prettier](https://prettier.io/docs/install)
```

Comment:
Should not have to manually install prettier as the Initialize project step will handle that.